src/bespokelabs/curator/prompter/prompter.py

```diff
@@ -45,6 +48,7 @@ def __init__(
             ]
         ] = None,
         response_format: Optional[Type[BaseModel]] = None,
+        backend: str = "openai",
```

Comment:
This makes sense to do now to give control and be able to test out our different backends, however @vutrung96 has made a good point that maybe we don't actually want to make this a choice that users know or think about.

Reply:
let's have this as None by default and have routing logic like we discussed.

if the user sets backend to "openai" or "litellm", then we'll respect their choices.

Reply:
Yep, agree

Reply:
i don't see this routing logic below?

if i set my model to claude, i also need to set my backend to "litellm"?